src/.formatted-files

```diff
@@ -0,0 +1,8 @@
+stat.h
+tweak.cpp
+tweak.h
+thinblock.cpp
+thinblock.h
+leakybucket.h
+parallel.cpp
+parallel.h
```

Comment:
nope.  As we discussed on slack, I do not feel it is "fair" to force people to manually format their contributions to different file.  So if the file is already using inconsistent formatting, I think we need to be pretty lenient.